buster/validators/base.py

```diff
@@ -1,44 +1,45 @@
 import logging
-from abc import ABC, abstractmethod
-from functools import lru_cache
 
 import pandas as pd
 
 from buster.llm_utils import cosine_similarity, get_openai_embedding
+from buster.validators.question_answer_validator import (
+    AnswerValidator,
+    DocumentsValidator,
+    QuestionValidator,
+)
 
 logger = logging.getLogger(__name__)
 logging.basicConfig(level=logging.INFO)
 
 
-class Validator(ABC):
+class Validator:
```

Comment:
Is there really a reason for this class now?

Reply:
i wonder the same thing, but it helps keep the code more organized